remote/style.css

```diff
@@ -106,4 +106,19 @@ body > .is-root-container,
 	border: 1px solid var(--wp--preset--color--foreground);
 	background-color: transparent;
 	color: var(--wp--preset--color--foreground);
+}
+
+.wp-block-tag-cloud.is-style-outline {
+	display: flex;
```

Comment:
What does this solve? Could we add it to the block?

Reply:
currently, they have inline-block instead of flex and that creates a 4px ish gap between items that you can't remove via margin/padding, so flex is better. It can be added to the block but it would be a very minor change that would make more sense in context with adding options to alter that spacing between the pills

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/38995

Reply:
These lines are in core now, so we can remove them from here.